--- origsrc/mysql-gui-tools-5.0r12/mysql-administrator/source/linux/Makefile.am	2006-12-05 16:46:58.000000000 -0600
+++ src/mysql-gui-tools-5.0r12/mysql-administrator/source/linux/Makefile.am	2008-05-16 14:09:11.484375000 -0500
@@ -45,10 +45,10 @@
 # mysql_administrator_bin_LDFLAGS=-pthread
 
 mysql_administrator_bin_LDADD=\
-	$(top_srcdir)/library/source/libadmin.a\
-	$(top_srcdir)/../mysql-gui-common/source/linux/libmacommongui.a\
-	$(top_srcdir)/../mysql-gui-common/library/base-library/source/.libs/libmysqlx.a\
-	$(top_srcdir)/../mysql-gui-common/library/sql-parser/source/.libs/libsqlparser.a\
-	$(top_srcdir)/../mysql-gui-common/library/utilities/source/.libs/libguiutil.a\
+	$(top_builddir)/library/source/libadmin.a\
+	$(top_builddir)/../mysql-gui-common/source/linux/libmacommongui.a\
+	$(top_builddir)/../mysql-gui-common/library/base-library/source/.libs/libmysqlx.a\
+	$(top_builddir)/../mysql-gui-common/library/sql-parser/source/.libs/libsqlparser.a\
+	$(top_builddir)/../mysql-gui-common/library/utilities/source/.libs/libguiutil.a\
 	@MYSQL_LIBS@ @GNOME_LIBS@ @UTIL_LIBS@ @PCRE_LIBS@
 
--- origsrc/mysql-gui-tools-5.0r12/mysql-administrator/source/util/Makefile.am	2006-10-18 04:57:48.000000000 -0500
+++ src/mysql-gui-tools-5.0r12/mysql-administrator/source/util/Makefile.am	2008-05-16 14:52:20.843750000 -0500
@@ -15,10 +15,10 @@
 
 
 mabackup_LDADD=\
-        $(top_srcdir)/library/source/libadmin.a\
-        $(top_srcdir)/../mysql-gui-common/library/base-library/source/.libs/libmysqlx.a\
-        $(top_srcdir)/../mysql-gui-common/library/utilities/source/.libs/libguiutil.a\
-        $(top_srcdir)/../mysql-gui-common/library/sql-parser/source/.libs/libsqlparser.a\
+        $(top_builddir)/library/source/libadmin.a\
+        $(top_builddir)/../mysql-gui-common/library/base-library/source/.libs/libmysqlx.a\
+        $(top_builddir)/../mysql-gui-common/library/utilities/source/.libs/libguiutil.a\
+        $(top_builddir)/../mysql-gui-common/library/sql-parser/source/.libs/libsqlparser.a\
         @MYSQL_LIBS@ @PCRE_LIBS@ @GLIB_LIBS@
 
 
--- origsrc/mysql-gui-tools-5.0r12/mysql-gui-common/configure.in	2007-03-29 09:46:22.000000000 -0500
+++ src/mysql-gui-tools-5.0r12/mysql-gui-common/configure.in	2008-05-16 13:40:18.562500000 -0500
@@ -165,7 +165,7 @@
 	[LUA_CFLAGS=-I$withval])
 AC_ARG_WITH(lua-libs,
 [  --with-lua-libs              directory where Lua libraries are located],
-        [LUA_LDFLAGS="$withval -llua -llualib"])
+        [LUA_LDFLAGS="-L$withval -llua50 -llualib50"])
 
 if test x"$LUA_CFLAGS$LUA_LDFLAGS" = "x"; then
   AC_CHECK_PROG(LUA_CFLAGS,lua-config,`lua-config --include`)
@@ -219,7 +219,7 @@
     AC_MSG_RESULT(not found)
   else
 # php-config --prefix ?
-    PHP_LIBS="/usr/lib/php5/lib/libphp5.a -L[$(php-config --extension-dir)] $PHP_LIBS"
+    PHP_LIBS="-lphp5lib $PHP_LIBS"
     PHP_CFLAGS=[$(php-config --includes)]
     ENABLED_MODULES="$ENABLED_MODULES -DENABLE_PHP_MODULES"
     AC_MSG_RESULT($PHP_LIBS)
@@ -328,15 +328,15 @@
 AC_MSG_CHECKING([for unicode support in pcre])
 ac_savelibs=$LIBS
 ac_savecpp=$CPPFLAGS
-LIBS=`pcre-config --libs`
-CPPFLAS=`pcre-config --cflags`
+LIBS="$PCRE_LIBS"
+CPPFLAGS="$PCRE_CFLAGS"
 AC_RUN_IFELSE([AC_LANG_PROGRAM([#include <pcre.h>], [
   char *err= NULL;
   int o;
   if (pcre_compile("(?:\\\\pL(?:\\\\pL|\\\\pN|\\\\pM|\\\\p{Pc}|\\\\p{Pd}|\\\\pS}|_)*)", PCRE_UTF8, &err, &o, NULL)) 
     return 0;
   return 1;
-])], [HAVE_UNICODE_PCRE=1 AC_MSG_RESULT(yes)], [HAVE_UNICODE_PCRE=0 AC_MSG_RESULT(no)])
+])], [HAVE_UNICODE_PCRE=1; AC_MSG_RESULT(yes)], [HAVE_UNICODE_PCRE=0; AC_MSG_RESULT(no)])
 CPPFLAGS=$ac_savecpp
 LIBS=$ac_savelibs
 AC_DEFINE(HAVE_UNICODE_PCRE, [], [whether pcre was compiled with unicode properties support])
@@ -349,6 +349,9 @@
 	DLMAKE="gcc -arch i386 -arch ppc -dynamiclib -flat_namespace -undefined suppress"
 elif test "$(uname -s)" = "SunOS"; then
 	DLMAKE="gcc -shared -mimpure-text"
+elif test "$host_os" = "cygwin"; then
+	DLEXT=.dll
+	DLMAKE="gcc -shared -Wl,--enable-auto-image-base"
 else
 	DLMAKE="gcc -shared"
 fi 
--- origsrc/mysql-gui-tools-5.0r12/mysql-gui-common/tools/grtsh/Makefile.am	2006-09-05 17:04:50.000000000 -0500
+++ src/mysql-gui-tools-5.0r12/mysql-gui-common/tools/grtsh/Makefile.am	2008-05-16 07:42:42.562500000 -0500
@@ -9,7 +9,7 @@
 if BUILD_NEWT
 newt_libs= $(top_builddir)/library/generic-runtime/source/.libs/libgrttextui.a\
 	$(top_builddir)/library/generic-runtime/newt/libmnewt.a \
-	@SLANG_LIB@ -ltermcap -lncurses
+	@SLANG_LIB@ -lncurses
 newt_flag= -DENABLE_TEXTUI
 endif
 
@@ -18,7 +18,7 @@
 endif
 
 if USE_READLINE
-rl_libs=-lncurses -lreadline -lhistory -ltermcap
+rl_libs=-lncurses -lreadline -lhistory
 grtsh_CPPFLAGS=-DUSE_READLINE 
 endif
 
